tests/codegen/vec-shrink-panik.rs

```diff
@@ -25,7 +25,8 @@ pub fn issue71861(vec: Vec<u32>) -> Box<[u32]> {
 
     // Call to panic_cannot_unwind in case of double-panic is expected
     // on LLVM 16 and older, but other panics are not.
-    // CHECK: filter
+    // old-CHECK: filter
+    // new-CHECK: cleanup
```

Comment:
```suggestion
    // old-CHECK: filter
```
As the comment says the `filter` only exists on old LLVM. It is optimized away in newer LLVM. Don't check for `cleanup`, that's just matching something completely different.